lib/commands/arm/hdinsight/hdinsight._js

```
@@ -1003,6 +1003,7 @@ hdInsightCommandLine.init = function(cli) {
     .option('-u, --uri <uri>', $('Script URI'))
     .option('-p, --parameters <parameters>', $('(Optional) Script parameters'))
     .option('-t, --nodeTypes <nodeTypes>', $('Specifies the nodes on which this cmdlet applies the action separated by ";". For example, "headnode;workernode" is a valid input.'))
+    .option('-a, --applicationName <applicationName>', $('Application name. When application name is specified, persistOnSuccess cannot be specified, and nodeTypes must contain only "edgenode"'))
```

I am not reasoning "why they are mutually exclusive".
I am asking is the validation for mutual exclusiveness done in the CLI? If not, then do you think it would be better to do it in the CLI rather than sending the request to the server and then failing based on the response received by the server. Simple validation check by the CLI will fail faster.
What do you think?